### PR TITLE
spotify: Run uninstaller only on uninstall

### PR DESCRIPTION
**Pull request by sitiom**
Fixes https://github.com/ScoopInstaller/Extras/discussions/8534

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/spotify.json [hunk 1]
```diff
@@ -25,7 +25,12 @@
         ]
     ],
     "uninstaller": {
-        "script": "Start-Process -Wait \"$dir\\Spotify.exe\" -ArgumentList '/Uninstall', '/Silent'"
+        "script": [
+            "# Run only on uninstall",
+            "if ($purge -ne $null) {",
```
Comment (issaclin32) [inline bucket/spotify.json:30]:
```suggestion
            "if ($purge) {",
```

**Comment by rashil2000** [comment]
Purge is a special case of uninstall. We should actually check for uninstallation and run the command thereafter. See https://github.com/ScoopInstaller/Extras/blob/edfc2d23b082ce74c5529eb1a017e373a69ba6cd/bucket/archwsl.json#L33 for example.

**Comment by sitiom** [comment]
> Purge is a special case of uninstall. We should actually check for uninstallation and run the command thereafter. See
> 
> https://github.com/ScoopInstaller/Extras/blob/edfc2d23b082ce74c5529eb1a017e373a69ba6cd/bucket/archwsl.json#L33
> 
> for example.

The Spotify uninstaller actually "purges" the config data. Just removing the spotify directory is already good enough.

**Comment by rashil2000** [comment]
When I manually run the Spotify uninstaller, it deletes data in both LocalAppData and AppData. This behaviour should be the same when I run `scoop uninstall spotify`, which the current script in the PR is not doing (it only does that when you pass the purge flag).

**Comment by sitiom** [comment]
So it has to match the installer's uninstall behavior 🤔

**Pull request merged**